CMSgov/mint-app
# [NOREF] Remove Dangling Comma in Favorite Cards

# NOREF

## Changes and Description

- Changed the way the model lead collaborators are accounted and joined together


|   |   | 
|---|---|
|Before|<img width="408" alt="image" src="https://github.com/CMSgov/mint-app/assets/2349702/ed0528e4-932e-4e3a-a3ab-6ad45b435834">|
|After|<img width="429" alt="image" src="https://github.com/CMSgov/mint-app/assets/2349702/14ce75ec-c1f0-4980-9633-97c1d6081bb0">|
 

## How to test this change

1. Seed server
2. Navigate to http://localhost:3005/models
3. Locate the model lead(s) in each favorite card
4. Check to see there is no more dangling comma


## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
